backend/src/main/resources/graphql/admin.graphqls

```diff
@@ -0,0 +1,61 @@
+# Queries and mutations that are used only by site admins
+extend type Query {
+  organizations: [Organization]
+}
+extend type Mutation {
+  createDeviceType(
+    name: String!
+    manufacturer: String!
+    model: String!
+    loincCode: String!
+    swabType: String!
+  ): DeviceType
+  updateDeviceType(
+    id: String!
+    name: String
+    manufacturer: String
+    model: String
+    loincCode: String
+    swabType: String
+  ): DeviceType
+  clearQueue: Int
+  addUser(
+    firstName: String
+    middleName: String
+    lastName: String!
+    suffix: String
+    email: String!
+    organizationExternalId: String!
+    role: Role!
+  ): User @requiredPermissions(allOf: ["MANAGE_USERS"])
```

Comment:
Could you remove this permission? This should site admins only, but I set it to the same level as `addUserToCurrentOrg` by mistake